tests/unit/horizontal-bar-test.js

```diff
@@ -48,3 +48,26 @@ test("correct behavior with null maxBarThickness and minBarThickness", function(
   assert.equal(component.get('minOuterHeight'), null,
     'minOuterHeight is null when minBarThickness is null');
 });
+
+test("X & Y titles coordinates are correct", function(assert){
+  assert.expect(4);
+
+  const component = this.subject({
+    graphicWidth: 100,
+    xTitleHorizontalOffset: 30,
+    graphicBottom: 50,
+    xTitleVerticalOffset: 10,
+    labelWidthOffset: 40,
+    graphicHeight: 70,
+    yTitleVerticalOffset: 10
+  });
+
+  assert.equal(component.get('xAxisPositionX'), 80,
+    'x position of X axis Title is correct');
+  assert.equal(component.get('xAxisPositionY'), 60,
+    'y position of X axis Title is correct');
+  assert.equal(component.get('yAxisPositionY'), -40,
+    'x position of Y axis Title is correct');
+  assert.equal(component.get('yAxisPositionX'), -25,
+    'y position of Y axis Title is correct');
+});
```

Comment:
You probably want to test when `graphWidth`, `xTitleHorizontalOffset` change, these values also change correspondingly.
For example now you set the `graphicWidth` to 200, assert the new values of these.